### PR TITLE
Hide Image.prototype.source

### PR DESCRIPTION
**Pull request by Hakerh400**
- [x] Updated changelog
- [x] Added test

### CODE REVIEW

**Comment by zbjornson** [comment]
Sorry for the delay, will try to review tomorrow!

File: src/Image.cc [hunk 1]
```diff
@@ -182,7 +184,7 @@ NAN_SETTER(Image::SetHeight) {
  * Get src path.
  */
 
-NAN_GETTER(Image::GetSource) {
+void Image::GetSource(const Nan::FunctionCallbackInfo<v8::Value> &info) {
```
Comment (zbjornson) [inline src/Image.cc]: I think this and line 227 should/could be `NAN_METHOD(Image::GetSource)` ?

Comment (Hakerh400) [reply src/Image.cc]: Rebased properly?

**Comment by Hakerh400** [comment]
@KenanSulayman

What do you mean? Which tests are broken by this PR? What you posted breaks all tests, since there is no more accessor property `source` on the image prototype. Are you sure you merged/fetched the updated files properly? Also, make sure to rebuild with node-gyp, so that C++ patches take effect.

**Comment by Hakerh400** [comment]
Unable to reproduce.

```console
D:\node_modules\canvas\examples> node resize.js
Resized and saved in 26ms
```

Can you please open a new issue if you believe that something is broken.

**Comment by KenanSulayman** [comment]
This is not meant to offend, but just because it works for you - and on your Windows installation - that does not mean it works for others.

Here's a recording of my bash session: https://asciinema.org/a/2VXF7vqdHS0Wk8FP30zfAvc7I.

**Comment by KenanSulayman** [comment]
Here's another assumption:  does the git build fetch the remote build no matter what?

**Comment by KenanSulayman** [comment]
Yes, that's the case. I can see the cause of this issue then.

**Comment by zbjornson** [comment]
> I can see the cause of this issue then.

@KenanSulayman are you all set here, or do you think something is broken?

**Pull request merged**